### PR TITLE
ci: Start supporting node.js 18

### PR DESCRIPTION
**Pull request by netroy**
Node.js 18 becomes the active LTS on 2022-10-25, as Node.js 16 goes into maintenance mode. 
https://github.com/nodejs/Release#release-schedule.

We'll should still continue to support Node.js 14 and 16 until they are End-of-life.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+18
```
Comment (ivov) [inline .nvmrc:1]: We're not mentioning anywhere that 18 is recommended for development. Should we? Maybe in the pnpm PR?

Comment (ivov) [reply .nvmrc:1]: Do we suggest `nvm`? Have not looked into the differences, but I have been using [`n`](https://github.com/tj/n) without issues. Perhaps to consult with the team.

File: .github/workflows/ci-postgres-mysql.yml [hunk 2]
```diff
@@ -15,12 +15,17 @@ jobs:
       DB_MYSQLDB_PASSWORD: password
       DB_POSTGRESDB_PASSWORD: password
 
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x]
```
Comment (ivov) [inline .github/workflows/ci-postgres-mysql.yml:20]: We're still supporting 14 so let's add it?

**Review by ivov — COMMENTED** [review]
Pending to check docker images.

File: .nvmrc (hunk 1 above)
Comment (netroy) [reply .nvmrc:1]: i've used both `n` and `nvm`, and they are both good for the job. but yes, we should ask the team first. I'll remove this file.

**Review by ivov — DISMISSED** [review]
Tested main and custom Docker images, confirmed working.

Pending conflict to resolve.

**Comment by netroy** [comment]
conflict resolved

**Review by ivov — APPROVED** [review]
Any reason not to keep `n8n-custom` image at 16 like the others?

Otherwise LGTM.

**Comment by netroy** [comment]
> Any reason not to keep `n8n-custom` image at 16 like the others?
> 
> Otherwise LGTM.

Using the `n8n-custom` image as a test for node 18. That way we can have the nightly image use node 18 for a while before we switch everything over to 18.

**Comment by janober** [comment]
Got released with `n8n@0.202.0`

**Pull request merged**